### ISSUE TITLE
We can use Ruffjs technology to display the flash screen correctly, but we cannot display the text content of the flash. May I ask why?

### ISSUE THREAD

**Issue by wangyang12611** [ruffle-rs/ruffle#12628]
### Describe the bug

We can use Ruffjs technology to display the flash screen correctly, but we cannot display the text content of the flash. May I ask why?As shown in the following figure,
![QQ图片20230808195716](https://github.com/ruffle-rs/ruffle/assets/20118702/68994a6a-f73d-45c8-8754-5a566e921142)

### Expected behavior

Display correct Chinese text in each process node box

### Affected platform

Online demo

### Operating system

win10

### Browser

chrome

**Comment by Lord-McSweeney** [comment]
#8390 might be relevant

**Comment by alephdfilms** [comment]
I don't believe that Ruffle supports Chinese text in Ruffle yet. Does this issue happen in other Flashes in Chinese? Maybe it's an ActionScript 3 issue?

**Comment by wangyang12611** [comment]
> I don't believe that Ruffle supports Chinese text in Ruffle yet. Does this issue happen in other Flashes in Chinese? Maybe it's an ActionScript 3 issue?

I see many other people's questions in the issue, which include Chinese characters that can be displayed and supported, but they are not supported. Is it Ruffle not supported or is it due to a lack of font

**Comment by kjarosh** [comment]
Closing this issue as there's no way to reproduce the issue (or even statically analyze it) without the SWF. Feel free to reopen or file a new one when the SWF is available
